debian/nymea-plugin-schneider-wallbox.install.in

```diff
@@ -0,0 +1,2 @@
+usr/lib/@DEB_HOST_MULTIARCH@/nymea/plugins/libnymea_integrationpluginschneiderWallbox.so
+schneiderWallbox/translations/*qm usr/share/nymea/translations/
```

Comment:
This would be the first plugin not following the naming convention of everything being lowercase... I see where it's coming from and it's probably not a big deal... In my experience this stuff will sooner or later break a regexp in some script or something somewhere (thinking docs generators and stuff)...